src/current/_includes/releases/v23.1/v23.1.17.md

```diff
@@ -0,0 +1,106 @@
+    1. The follower's [closed timestamp](https://www.cockroachlabs.com/docs/v23.1/architecture/transaction-layer#closed-timestamps) has advanced past the transaction commit timestamp.
+    1. The rangefeed attempts to push the transaction to a new timestamp (at least 10 seconds after the transaction began).
+    1. This may cause the rangefeed to prematurely emit a checkpoint before emitting writes at lower timestamps, which in turn may cause the [changefeed](https://www.cockroachlabs.com/docs/v23.1/change-data-capture-overview#how-does-an-enterprise-changefeed-work) to drop these events entirely, never emitting them.
+
+    [#118415][#118415]
+- Fixed a bug introduced in v22.2 where queries issued by the TTL jobs did not use optimal plans. [#118496][#118496]
+- Fixed a bug introduced in v23.1.0 where a decommissioning replica that is part of a mis-replicated range could get stuck on a rebalance operation that was falsely determined to be unsafe. [#118437][#118437]
+- Fixed a bug introduced in v22.2 where adding multiple columns with unique constraints in a single statement could result in the error "secondary index for backfill contains physical column not present in source primary index " [#118137][#118137]
+- Fixed a bug where "unable to get CPU capacity" errors could be logged every 10 seconds when running outside of a CPU `cgroup`. [#118670][#118670]
+- Fixed a bug where an `AUTO CREATE STATS` job could cause an internal system table to grow, and this could cause slow queries related to the job system. [#118944][#118944]
+- Fixed a bug that caused a confusing error when a sequence name allocated by `SERIAL` conflicted with an existing type name. [#118948][#118948]
+- Fixed a bug where an `ALTER PRIMARY KEY` query could fail with the error `non-nullable column {x} with no value! Index scanned ..` when validating a recreated secondary index. [#118972][#118972]
+- Fixed a bug introduced in v23.1 that could cause the internal error `attempting to append refresh spans after the tracked timestamp has moved forward` to be logged in some cases when using virtual tables such as`crdb_internal.system_jobs`. [#119186][#119186]
+- Fixed a bug in the `crdb_internal.leases` table where a deadlock in the leasing system could cause a node to become unavailable. [#119374][#119374]
+- Fixed a bug introduced in v23.1.15 where a rangefeed-resolved timestamp could get stuck after a range merge if the range cache thinks the barrier spans multiple ranges. The log message `pushing old intents failed: range barrier failed, range split` is constantly emitted. [#119543][#119543]
+- Fixed a rare panic that could happen during a `pg_dump` import that contains a subquery in one of its argume nts, such as a function like `SELECT addgeometrycolumn(...)`. Now, attempting to import a `pg_dump` with a function that has a subquery in one of its arguments results in an error rather than a panic. [#118611][#118611]
```

Comment:
```suggestion
- Fixed a rare panic that could happen during a `pg_dump` import that contains a subquery in one of its arguments, such as a function like `SELECT addgeometrycolumn(...)`. Now, attempting to import a `pg_dump` with a function that has a subquery in one of its arguments results in an error rather than a panic. [#118611][#118611]
```